examples/dnspolicy/aws-dns-provider-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-credentials
+  namespace: ${DNSPOLICY_NAMESPACE}
+type: "kuadrant.io/aws"
+stringData:
+  AWS_ACCESS_KEY_ID: ${KUADRANT_AWS_ACCESS_KEY_ID}
+  AWS_SECRET_ACCESS_KEY: ${KUADRANT_AWS_SECRET_ACCESS_KEY}
+  AWS_REGION: ${KUADRANT_AWS_REGION}
```

Comment:
missing `ZONE_ID_FILTER` and `ZONE_DOMAIN_FILTER` ?

Reply:
These are no longer there. Instead we rely on the credential to filter down via permissions. Rather than having it in two places, suggestion is to create a secret with a credential that has access to the zones you want kuadrant to manage

Reply:
going to do a quick check for any other refs to that

Reply:
Sorry, I am confused. Do you mean they are optional or should not be there?

I see those two fields being added multiple times:

* https://github.com/Kuadrant/kuadrant-operator/pull/793/files#diff-676c95362ed2e144f27da53f2abc58119334a8f650ddbb61b93d2ff18ae8ef07R126
* https://github.com/Kuadrant/kuadrant-operator/pull/793/files#diff-0b71d4d09687455f24aaa3a2326bed45baa748fdfff795171148e2df85a066a8R52
* https://github.com/Kuadrant/kuadrant-operator/pull/793/files#diff-dd9adc823d5cb403f5710f23e6624096dd323f422723e7672b0f76c51385d849R242
* https://github.com/Kuadrant/kuadrant-operator/pull/793/files#diff-dd9adc823d5cb403f5710f23e6624096dd323f422723e7672b0f76c51385d849R262

Reply:
Yes. I noticed these also. Sorry took this PR over so was unsure. They are gone. So I will remove from those docs also

Reply:
ok should all be gone now @eguzki I also added an example AWS policy to set permissions